src/chrome/content/rules/QQ.xml

```diff
@@ -281,7 +278,12 @@
 	<target host="i.mail.qq.com" />
 	<target host="res.mail.qq.com" />
 	<target host="rl.mail.qq.com" />
+	<target host="map.qq.com" />
+		<exclusion pattern="^http://map\.qq\.com/(?!api/)" />
+		<test url="http://map.qq.com/api/js" />
```

Comment:
`http://map.qq.com/api/js` seems to request hard-coded http urls, and https-everywhere won't have chance to rewrite the requests before they are blocked 	except Tor browser...